terraform/modules/frontdoor/url_rewriter.js

```diff
@@ -0,0 +1,54 @@
+function url_rewriter(event) {
+    const exceptions = ["landlord", "local-authority", "sign-out", "error", "assets"];
+    const request = event.request;
+    let pathSegments = request.uri.split('/');
+    let domainSegmentIndex;
```

Comment:
The example [here](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/example_cloudfront_functions_kvs_conditional_read_section.html) hard codes the index of the segment they want to change, but I thought we might need this to work whether we included the `https://` or not.

Reply:
I think I prefer this anyway given we know what the domains we're expecting should look like.